server/src/challenge/challenge.e2e-spec.ts

```diff
@@ -133,7 +133,7 @@ describe('ChallengeModule E2E', () => {
       const chaldto: ChallengeDto = {
         id: '12345',
         name: 'test',
-        location: 'ENG_QUAD',
+        location: LocationTypeDto['EngQuad' as keyof typeof LocationTypeDto],
```

Comment:
Not entirely sure what's happening here, but there is a high likelihood you can simplify this